src/components/RecipeBox.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+export class RecipeBox extends Component {
+	render() {
+		return (
+			<div className="recipeBox">
```

Comment:
basically more common approach is `css-class-name` but it depends on decision made by the team